packages/client/src/constants.ts

```diff
@@ -9,6 +9,7 @@ export const CREDENTIAL_SOURCE_FILE = "file"
 export const CREDENTIAL_SOURCE_ENV = "env"
 export const CREDENTIAL_SOURCE_DIRECT = "direct"
 export const CREDENTIAL_SOURCE_OPTIONS = [CREDENTIAL_SOURCE_FILE, CREDENTIAL_SOURCE_ENV, CREDENTIAL_SOURCE_DIRECT]
+export const CREDENTIAL_SOURCE_OPTIONS_CONST = [CREDENTIAL_SOURCE_FILE, CREDENTIAL_SOURCE_ENV, CREDENTIAL_SOURCE_DIRECT] as const;
```

Comment:
```suggestion
export const CREDENTIAL_SOURCE_OPTIONS = [CREDENTIAL_SOURCE_FILE, CREDENTIAL_SOURCE_ENV, CREDENTIAL_SOURCE_DIRECT] as const;
```
No need to include CONST in the name.

Reply:
Noted